src/layouts/ResourceRoom/ResourceRoom.tsx

```diff
@@ -80,40 +92,72 @@ const EmptyResourceRoom = () => {
     }
   }, [errorToast, isError])
 
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
   return (
-    <SiteViewLayout>
-      <Box as="form" w="full">
+    <SiteViewLayout
+      divider={undefined} // to remove bottom divider line
```

Comment:
should we prefer using fragments for this use case? ie, 
```ts
<>
<SiteViewLayout />
<Modal />
</>
```

this avoids having to sidestep the divider

Reply:
i think this^ looks better instead of having `divider = {undefined}` - wdyt?

Reply:
eh I think got misunderstanding here bah - this components  for this empty state will be shown below the entire page leh, unless we redefine what `SiteViewLayout` does which I dont feel should be the case here

Reply:
hmm i can't repro this! feel free to ping me to pair on it

Reply:
fixed